source/frontend/src/models/generate/acquisition/GenerateExpropriationForm1.ts

```diff
@@ -22,7 +22,6 @@ export class Api_GenerateExpropriationForm1 extends Api_GenerateExpropriationFor
     impactedProperties = [],
```

Comment:
the field on line 10 should also be removed.

Reply:
also the prop on line 16

Reply:
also there is a usage of ExpropriationNoticeServedDate in the file ExpropriationForm1YupSchema.ts